### DISCUSSION TITLE
[287390] 地铁：最后的曙光 Redux | 本体

### DISCUSSION THREAD

**Discussion by a666ab** [pjy612/SteamManifestCache#548] [Share]
appid:287390
 keys:
 ```
 287391;b83248490dbb4df7788d5976584e6829597922e10fb4a9ab537e4da04c52b61b
287392；b31b8b47d359f387ffb0f435de1b127ab2a3dfcfd16304b1d18efff394f3c7f5
 ```
[[287390] 地铁：最后的曙光 Redux.zip](https://github.com/pjy612/SteamManifestCache/files/14657992/287390.Redux.zip)
 email;1278473419@qq.com

**Comment by pjy612** [comment]
* 似乎缺主程 https://steamdb.info/app/287390/depots/

**Reply by a666ab** [reply]
这个主程序是什么，我也是这两天刚接触。密钥跟清单不都在吗？是.lua后缀的文件吗？我下载了别人的看了看，似乎一样。不太懂，请教一下，麻烦了。

**Reply by pjy612** [reply]
缺 287392 的密钥和清单

**Reply by a666ab** [reply]
好的，已修改
